CRM/Activity/Import/Parser/Activity.php

```diff
@@ -264,7 +264,7 @@ public function import($onDuplicate, &$values) {
     $session = CRM_Core_Session::singleton();
     $dateType = $session->get('dateTypes');
     if (!isset($params['source_contact_id'])) {
-      $params['source_contact_id'] = $session->get('userID');
+      $params['source_contact_id'] = CRM_Core_Session::singleton()->getLoggedInContactID();
```

Comment:
Yeah - I was trying to do a load of them - there might be one or 2 that could be better